lib/grizzly/transports/DTLS.ex

```diff
@@ -54,23 +64,31 @@ defmodule Grizzly.Transports.DTLS do
 
   # Erlang/OTP >= 23.2
   def parse_response(
-        {:ssl, {:sslsocket, {:gen_udp, {{ip, _}, _}, :dtls_gen_connection}, _}, bin_list},
+        {:ssl, {:sslsocket, {:gen_udp, {{ip, port}, _}, :dtls_gen_connection}, _}, bin_list},
         opts
```

Comment:
Doesn't the ignored part of the tuple above the destination? I.e. `{ip, port}` is the source and the `_` right after it could be used as the destination. If so, then the pattern match could be simplified to `{:gen_udp, {src, dest}}` and I think that simplifies the code a little.